components/src/dynamo/common/tests/multimodal/test_embedding_transfer.py

```
@@ -0,0 +1,110 @@
+    LocalEmbeddingSender,
+    NixlEmbeddingReceiver,
+    NixlEmbeddingSender,
+    NixlPersistentEmbeddingReceiver,
+    NixlPersistentEmbeddingSender,
+)
+
+logger = logging.getLogger(__name__)
+
+
+async def benchmark(
+    sender, receiver, tensors=[torch.randn(256, 8 * 1024) for _ in range(30)]
```

Ruff (0.14.14)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)